scripts/l2-bridge/l2-bridge-deploy-l1-multicall.sh

```diff
@@ -6,33 +6,42 @@ set -a
 source $(pwd)/.env
 set +a
 
-cd "$(pwd)/multicall"
-forge build
+# Path to the .env.bridge file
+ENV_FILE=$(pwd)/.env.bridge
 
+echo "Checking if multicall contract is already deployed..."
 CHAIN_ID=$(cast chain-id --rpc-url "${L1_RPC_URL}")
-
-DEPLOYED_URL=$(curl https://raw.githubusercontent.com/mds1/multicall/main/deployments.json | grep -A 3 "\"chainId\": ${CHAIN_ID},"  | grep "\"url\":" | awk -F'"' '{print $4}')
+DEPLOYED_URL=$(curl -s https://raw.githubusercontent.com/mds1/multicall/main/deployments.json | jq -r ".[] | select(.chainId == $CHAIN_ID) | .url")
 if [ ! -z "${DEPLOYED_URL}" ]; then
     # Extract the Ethereum address from the URL
+    echo "Extracting multicall contract address from the deployed URL..."
     DEPLOYED_ADDRESS=$(echo "$DEPLOYED_URL" | grep -oE '0x[a-fA-F0-9]{40}')
     if [ -n "$DEPLOYED_ADDRESS" ]; then
         echo "Contract is already deployed. Exiting."
-        # Update the .env file with the deployed address
-        sed -i.bak "s/^NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=.*/NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=${DEPLOYED_ADDRESS}/" ../.env
-        rm ../.env.bak
```

Comment:
i am a bit confused. why simply changing these two lines to .env.bridge won't work?

Reply:
actually it should work, and other changes are the improvements and logs

Reply:
got it. yeah i think we shouldn't touch .env for bridge related scripts.

it's cleaner to just use .env.bridge